powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/v3/ActivationServiceBehavior.java

```diff
@@ -1209,7 +1245,7 @@ public CommitActivationResponse commitActivation(String activationId, String ext
 
         // Change activation state to ACTIVE
         activation.setActivationStatus(io.getlime.security.powerauth.app.server.database.model.ActivationStatus.ACTIVE);
-        activationHistoryServiceBehavior.saveActivationAndLogChange(activation, externalUserId);
+        activationHistoryServiceBehavior.saveActivationAndLogChange(activation, externalUserId, AdditionalInformation.ACTIVATION_VERSION_CHANGED);
```

Comment:
Z tohohle jsem hodně nervózní, protože jsem k tomu nenašel test (který bych upravil) ani se mi nepodařilo žádný shodit, aby mi to pomohlo nějaký najít.

Reply:
V issue se píše, že v commit activation se nastavuje verze, ale to tady v kódu nevidím. To se počítá ta změna z prepare activation?

Reply:
End-to-end tests for PowerAuth are here: https://github.com/wultra/powerauth-tests. They require extra setup (e.g. Next Step component), so I can run the tests myself. We do have technical debt in unit tests on the `powerauth-server` component.

As for the version, yes, sorry, the version is set in the `prepare` phase, I will update the issue.

Reply:
Understood, I moved the reason from _commit_ to _prepare_ activation.